### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by lieut-data**
#### Summary
Update README.md to reflect product direction.

#### Ticket Link
None.

### CODE REVIEW

**Review by sbishel — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,25 +1,14 @@
 # Mattermost MS Teams Plugin
 
-This plugin integrates MS Teams with Mattermost by providing automated syncing of messages from Mattermost to MS Teams and vice versa.  
+The plugin enables you to break through siloes in a mixed Mattermost and Teams environment by forwarding real-time chat notifications from Teams to Mattermost.
```
Comment (cwarnermm) [inline README.md:3]:
```suggestion
The plugin enables you to break through siloes in a mixed Mattermost and Microsoft Teams environment by forwarding real-time chat notifications from Microsoft Teams to Mattermost.
```

File: README.md [hunk 2]
```diff
@@ -1,25 +1,14 @@
 # Mattermost MS Teams Plugin
```
Comment (cwarnermm) [inline README.md:1]:
```suggestion
# Mattermost Microsoft Teams Plugin
```

File: README.md [hunk 3]
```diff
@@ -1,25 +1,14 @@
...
-- Deletion of MS Teams messages is synced with Mattermost but it's not vice versa.
-- Sync posts containing markdown and attachments.
-
-    ![image](https://user-images.githubusercontent.com/77336594/226587339-050c35da-a0f1-47db-a15f-f8d5f59bf8cd.png)
-    ![image](https://user-images.githubusercontent.com/77336594/226587366-2c4231bc-1aa2-42c4-b692-bd4441c71c34.png)
-    ![image](https://user-images.githubusercontent.com/77336594/226588263-a7915e4d-d9ae-4294-9134-326628febdfc.png)
-    ![image](https://user-images.githubusercontent.com/77336594/226588309-3202b78f-d87d-439c-967b-25ba8ed328c9.png)
-
-- Sync reactions on posts.
+- Connect to an MS Teams account using the OAuth2 flow.
```
Comment (cwarnermm) [inline README.md:9]:
```suggestion
- Connect to an MicrosoftS Teams account using the OAuth2 flow.
```

**Review by cwarnermm — APPROVED** [review]
Replaced MS Teams with Microsoft Teams to follow third-party branding expectations.

File: README.md (hunk 3 above)
Comment (lieut-data) [reply README.md:9]:
```suggestion
- Connect to an Microsoft Teams account using the OAuth2 flow.
```

**Review by JulienTant — APPROVED** [review]

**Pull request merged**